dirty2.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+)
+
+func zFoo(arg string) int {
+	return 1
+	return 1
+}
+
+func complexFunc5(r io.Reader) error {
```

Comment:
__[staging][gometalinter]__
* [dupl] duplicate of dirty.go:22-89
* [dupl] duplicate of dirty2.go:50-82
* [gocyclo] cyclomatic complexity 11 of function complexFunc5() is high (> 10)
* [deadcode] complexFunc5 is unused

<!-- Comment by [SideCI](https://sideci.com) -->